staging/src/k8s.io/kube-aggregator/pkg/apiserver/apiserver.go

```diff
@@ -301,14 +304,18 @@ func (s *APIAggregator) AddAPIService(apiService *v1.APIService) error {
 		proxyPath = "/api"
 	}
 
+	restConfigProvider, err := util.NewDynamicRestConfigProvider(s.certKeyContentProvider, s.egressSelector, s.proxyTransport, apiService.Spec.InsecureSkipTLSVerify, apiService.Spec.Service.Name + "." + apiService.Spec.Service.Namespace + ".svc", apiService.Spec.CABundle)
```

Comment:
we need to stop it when `RemoveAPIService` was called.

Reply:
oh good point... but right now when I removed the Run method it's actually just implementing Notifier, so there's nothing to stop anymore, correct?

Reply:
hmm okay, but we still use up the memory we should clean up, wdyt?